crates/rolldown/src/stages/link_stage/wrapping.rs

```diff
@@ -103,7 +103,7 @@ impl LinkStage<'_> {
         );
       }
 
-      if !module.star_exports.is_empty() {
+      if module.has_star_export() {
```

Comment:
Most of modules doesn't have `star_export`, this cause a `o(1)` check method become `o(n)` check which is called on every modules.